Palo Alto Networks/paloalto-cortex/ingest/parser.yml

```
@@ -0,0 +1,37 @@
+name: cortex-xdr
+pipeline:
+  - name: parsed_event
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: setter
+    description: ''
+    filter: ''
+    external: null
+stages:
+  setter:
+    actions:
+      - set:
+          cortexxdr.shost: '{{parsed_event.message.shost}}'
```

Please use [ECS fields](https://www.elastic.co/guide/en/ecs/8.4/ecs-event.html) as the base of your taxonomy.
For example, here, [`host.name`](https://www.elastic.co/guide/en/ecs/8.4/ecs-host.html#field-host-name) is more suitable.
```suggestion
          host.name: '{{parsed_event.message.shost}}'
```